row_050_2/row_050_2_unsafe.py

```
@@ -0,0 +1,344 @@
+            dt.config(text="Successful", bg="GREEN", fg="Dark GREEN")
+        except:
+            dt.config(text="Error", fg="RED", font="20")
+        finally:
+            cm.close()
+
+
+    def OnVsb(*args):
+        listbox1.yview(*args)
+        listbox2.yview(*args)
+        listbox3.yview(*args)
+        listbox4.yview(*args)
```

Ruff (0.14.1)

`listbox4` may be undefined, or defined from star imports

(F405)